orm/ai-sdk-nextjs/README.md

```
@@ -0,0 +1,94 @@
+
+```bash
+npx try-prisma@latest --template orm/ai-sdk-nextjs
+```
+
+Then, navigate into the project directory:
+
+```bash
+cd ai-sdk-nextjs
+```
+
+<details><summary><strong>Alternative:</strong> Clone the entire repo</summary>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: details

(MD033, no-inline-html)